hack/release.sh

```diff
@@ -41,7 +41,12 @@ PUSH=
 LATEST=
 CSI_IMAGE_NAME=
 SYNCER_IMAGE_NAME=
-VERSION=$(git log -1 --format=%h)
+if [[ "$(git rev-parse --abbrev-ref HEAD)" =~ "master" ]]; then
+  VERSION="$(git log -1 --format=%h)"
+else
+  VERSION="$(git describe --always 2>/dev/null)"
+fi
+GIT_COMMIT="$(git log -1 --format=%h)"
```

Comment:
For `GIT_COMMIT` we can use full commit ID

Reply:
Fixed by changing command to `GIT_COMMIT="$(git log -1 --format=%H)"`